doc/operations/recovery.rst

```diff
@@ -16,10 +16,16 @@ The recovery procedure consists of two phases:
 Establishing a Recovered Public Network
 ---------------------------------------
 
-To initiate the first phase of the recovery procedure, one or several nodes should be started with the ``recover`` option:
+To initiate the first phase of the recovery procedure, one or several nodes should be started with the ``Recover`` command in the `cchost` config file:
 
 .. code-block:: bash
 
+    $ cat /path/to/config/file
+      ...
+      "command": {
+        "type": "Recover",
+        "start":
```

Comment:
Good spot! One issue with this sample:

```suggestion
        "recover":
```

We duplicate the command, as both `command.type` and the name of a field containing command-specific params - see `recover_config.json` for a sample.

Reply:
Aha! Yes, that wasn't obvious from the actual file. I'll add the whole `recover` element and perhaps add a link to the sample config.